editoast/src/schema/errors.rs

```diff
@@ -35,6 +35,7 @@ pub enum InfraErrorType {
     MissingBufferStop {
         endpoint: Endpoint,
     },
+    NodeEndpointsNotUnique,
```

Comment:
Do you think this kind of error could be related to anything else than a switch? If not, should this variant contain the word switch in it?

Reply:
We want to actively replace switch occurrences with node everywhere (code and interface) (checkout https://github.com/osrd-project/osrd/issues/5346). To follow this approach and avoid having to refactor this part, I've used node directly.